src/Parser.js

```diff
@@ -705,7 +645,9 @@ export default class Parser {
     parseSizeGroup(optional: boolean): ?ParseNode<"size"> {
         let res;
         let isBlank = false;
-        if (!optional && this.fetch().text !== "{") {
+        // don't expand before parseStringGroup
+        this.gullet.consumeSpaces();
+        if (!optional && this.gullet.future().text !== "{") {
```

Comment:
Is the comment referring to the `if` block?

Reply:
@kevinbarabash Yes, but also to `this.gullet.consumeSpaces();`, on why it's used instead of `this.consumeSpaces();`.